### ISSUE TITLE
Cannot build the project dependencies update

### ISSUE THREAD

**Issue by elturpin** [szymmis/vite-express#99]
Hi !

I wanted to add something new in the project but unfortunately, I cannot build the project anymore.

This is the error log:
```
src/main.ts → dist...
[!] RollupError: Expected ',', got '{' (Note that you need plugins to import files that are not JavaScript)
src/main.ts (8:12)
 6: import path from "path";
 7: import pc from "picocolors";
 8: import type { ViteDevServer } from "vite";
                ^
 9: 
10: type ViteConfig = {
RollupError: Expected ',', got '{'
    at error (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/parseAst.js:279:30)
    at nodeConverters (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/parseAst.js:2147:9)
    at convertNode (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/parseAst.js:1032:12)
    at convertProgram (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/parseAst.js:1023:48)
    at Object.parseAstAsync (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/parseAst.js:2213:20)
    at Module.tryParseAsync (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/rollup.js:14936:21)
    at Module.setSource (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/rollup.js:14517:35)
    at ModuleLoader.addModuleSource (/home/elturpin/Projets/vite-express/node_modules/rollup/dist/shared/rollup.js:18827:13)
```

Rollup does not manage to parse typescript even though the rollup plugin is there.
Using git bisect, I found that i cannot build the project since the commit "Update development dependencies" where a lot of dependencies are updated.

I tried to create a dummy project with only rollup typescript and one *hello world* file but the build worked perfectly with the same version of the packages.

I also tried to rollback the update only on typescript(4.9.4), tslib(2.4.1), rollup(3.7.3) and @rollup/plugin-typescript(10.0.1), and then the build was possible.

Any Idea ??

My env:
+ Node : 18.18.2
+ NPM : 10.2.5

**Comment by szymmis** [comment]
Hi @elturpin, there is indeed some problem with building the package that was not present in CI so it missed completely over my head. I need to look at it, there is something fishy with the `rollup` because as you said, there is a typescript plugin but it refuses to compile.

Thanks for bringing that up, I'll take a look at that as soon as I have time.

**Comment by elturpin** [comment]
Hi @szymmis 

I made some investigation and finally manage to reproduce the problem in a minimal repo : https://github.com/elturpin/test-rollup-ts-picocolors

The problem seems to come from the picocolors package. I have no idea why but the same compilation error occurs when picocolors is imported in a file. If the package is not imported, no problem ...

I'll try to investigate more when i'll have some free time.

**Comment by szymmis** [comment]
Should be fixed by #101.
